translations/terminal_tutorial/terminal_tutorial.ger.md

```diff
@@ -0,0 +1,389 @@
+# [Zu diesem Projekt beitragen](https://syknapse.github.io/Contribute-To-This-Project/)
+
+![image info](/favicon.png)
+
+> Logo mit :sparkling_heart: erstellt von [CandidDeer](https://github.com/CandidDeer)
+
+[![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)][twit]
+
+[![Discord](https://badgen.net/discord/online-members/tWkvS4ueVF?label=Join%20Our%20Discord%20Server&icon=discord)](https://discord.gg/tWkvS4ueVF 'Join our Discord server!')
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://syknapse.github.io/Contribute-To-This-Project/)
+[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://syknapse.github.io/Contribute-To-This-Project/)
+
+---
+
+> ## Ausschreibung
+>
+> Willst Du ein Maintainer dieses Projekts werden und helfen, es am Laufen zu halten? Bei Interesse lies den [Maintainers Guide](/translations/maintainer_guide/maintainer_guide.ger.md) und sende mir eine PN bei [Twitter](https://twitter.com/Syknapse).
+
+---
+
+### Schnellzugriff
+
+#### Übersicht
+
+- [Ausschreibung](#ausschreibung)
+- [Einleitung](#einleitung)
+- [Für wen ist es geeignet?](#für-wen-ist-es-geeignet)
+- [Warum sollte ich mitmachen?](#warum-sollte-ich-mitmachen)
+- [Was werde ich beitragen?](#was-werde-ich-beitragen)
+- [Übersetzungen](#übersetzungen)
+- [Setup](#setup-)
+- [Nächste Schritte](#nächste-schritte)
+- [Danke](#danke)
+
+#### Steps
+
+- [Zu diesem Projekt beitragen](#zu-diesem-projekt-beitragen)
+  - [Schnellzugriff](#schnellzugriff)
+    - [Übersicht](#übersicht)
+    - [Steps](#steps)
+  - [Einleitung](#einleitung)
+    - [Ziele](#ziele)
+    - [Für wen ist es geeignet](#für-wen-ist-es-geeignet)
+    - [Warum sollte ich mitmachen?](#warum-sollte-ich-mitmachen)
+    - [Was werde ich beitragen?](#was-werde-ich-beitragen)
+    - [Übersetzungen](#übersetzungen)
+    - [Setup! :)](#setup-)
+    - [Beitragen](#beitragen)
+      - [Schritt 1: Fork dieses Repository](#schritt-1-fork-dieses-repository)
+      - [Schritt 2: Clone das Repository](#schritt-2-clone-das-repository)
+      - [Schritt 3: Erstelle einen Branch](#schritt-3-erstelle-einen-branch)
+      - [Schritt 4: Öffne die Datei index.html](#schritt-4-öffne-die-datei-indexhtml)
+      - [Schritt 5: Kopiere die Karten-Vorlage](#schritt-5-kopiere-die-karten-vorlage)
+      - [Schritt 6: Änderungen eintragen](#schritt-6-änderungen-eintragen)
+      - [Step 7: Commit Deine Änderungen](#step-7-commit-deine-änderungen)
+      - [Schritt 8: Push Deine Änderungen zu GitHub](#schritt-8-push-deine-änderungen-zu-github)
+      - [Schritt 9: Erstelle einen PR(Pull Request)](#schritt-9-erstelle-einen-prpull-request)
+      - [Schritt 10: Feier](#schritt-10-feier)
+    - [Nächste Schritte](#nächste-schritte)
+    - [Danke](#danke)
+    - [Top 100 Beitragende](#top-100-beitragende)
```

Comment:
why are there too many steps?

Reply:
Sorry for that, was an Extension auto-generating the TOC, should be fixed now.